deployment/terraform/participant/main.tf

```diff
@@ -232,6 +244,28 @@ resource "azurerm_storage_account" "inbox" {
   account_kind             = "StorageV2"
 }
 
+resource "azurerm_storage_account" "shared" {
+  name                     = "${var.prefix}${var.participant_name}shared"
+  resource_group_name      = azurerm_resource_group.participant.name
+  location                 = azurerm_resource_group.participant.location
+  account_tier             = "Standard"
+  account_replication_type = "LRS"
+  account_kind             = "StorageV2"
+  static_website {}
```

Comment:
why `static_website`? This will create a public HTTP endpoint, but does not seem to be needed here